File: java/getting-localized-content/GetLocalizedItems.java
// DocSection: getting_localized_content_language
import com.kenticocloud.delivery;

DeliveryClient client = new DeliveryClient("975bf280-fd91-488c-994c-2f04416e5ee3");

List<NameValuePair> params = DeliveryParameterBuilder.params().language("es-ES").build();

// Create strongly typed models according to https://developer.kenticocloud.com/docs/strongly-typed-models
ArticleItem item = client.getItem("on_roasts", ArticleItem.class, params);
// EndDocSection

File: ruby/getting-localized-content/GetLocalizedItems.rb
# DocSection: getting_localized_content_language
require 'delivery-sdk-ruby'

delivery_client = KenticoCloud::Delivery::DeliveryClient.new project_id: '975bf280-fd91-488c-994c-2f04416e5ee3'
delivery_client.item('on_roasts')
               .language('es-ES')
               .execute do |response|
                 item = response.item
               end
# EndDocSection